nixos/modules/module-list.nix

```
@@ -2,6 +2,7 @@
   ./config/appstream.nix
   ./config/console.nix
   ./config/debug-info.nix
+  ./config/flakes.nix
```

```suggestion
  ./config/nix-flakes.nix
```
all other things include nix so we might as well